`amp-skimlinks` tests are flaky

```
DESCRIBE => amp-skimlinks
  DESCRIBE => Basic features
    DESCRIBE =>  
      IT => Should send the page impression tracking request
        ✗ Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

DESCRIBE => amp-skimlinks
  DESCRIBE => Basic features
    DESCRIBE =>  
      IT => should send NA-tracking on non-merchant link click 
        ✗ Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```
Failures from the past few hours:

https://travis-ci.org/ampproject/amphtml/jobs/528860786#L1940-L1944
https://travis-ci.org/ampproject/amphtml/jobs/527886895#L1533-L1539
https://travis-ci.org/ampproject/amphtml/jobs/527096273#L1909-L1914
https://travis-ci.org/ampproject/amphtml/jobs/528755387#L377-L381

/to @sparhami, who is build cop this week

I'm seeing more `skimlinks` tests failing on `master` today.

```
DESCRIBE => amp-skimlinks
  DESCRIBE => Affiliate unknown links
    DESCRIBE =>  
      IT => Should send unknown links to waypoint
        ✗ AssertionError: Target cannot be null or undefined.
            at http://localhost:9876/home/travis/build/ampproject/amphtml/test/integration/test-amp-skimlinks.js:248:29
```
https://travis-ci.org/ampproject/amphtml/jobs/529288928#L1940-L1945
https://travis-ci.org/ampproject/amphtml/jobs/529101437#L1940-L1945

Seems like there's a more widespread problem with this suite?

--- a/test/integration/test-amp-skimlinks.js
+++ b/test/integration/test-amp-skimlinks.js
@@ -116,7 +116,7 @@ describe.skip('amp-skimlinks', function() {
 
     // TODO(alanorozco): Unskip on firefox
     const itSkipFirefox = (desc, cb) =>
-        it.configure().skipFirefox().run(desc, cb);
+      it.configure().skipFirefox().run(desc, cb);
 
     // TODO(alanorozco): Unskip on firefox
     itSkipFirefox('should send NA-tracking on non-merchant link click ', () => {

--- a/test/integration/test-amp-skimlinks.js
+++ b/test/integration/test-amp-skimlinks.js
@@ -41,7 +41,8 @@ function clickLinkAndNavigate_(doc, selector) {
   }
 }
 
-describe('amp-skimlinks', function() {
+// TODO(@slocka) Disabled due to #22154, re-enable
+describe.skip('amp-skimlinks', function() {
   const setupBasic = {
     extensions: ['amp-skimlinks'],
     body: `
@@ -78,8 +79,7 @@ describe('amp-skimlinks', function() {
       return browser.waitForElementBuild('amp-skimlinks');
     });
 
-    // TODO(@slocka) Disabled due to #22154, re-enable
-    it.skip('Should send the page impression tracking request', () => {
+    it('Should send the page impression tracking request', () => {
       return RequestBank.withdraw('pageTrackingUrl').then(req => {
         const regex = /^\/track\.php\?data=([^&]*)&?.*$/;
         const match = regex.exec(req.url);
@@ -94,8 +94,7 @@ describe('amp-skimlinks', function() {
       });
     });
 
-    // TODO(@slocka) Disabled due to #22154, re-enable
-    it.skip('Should send the links impression tracking request', () => {
+    it('Should send the links impression tracking request', () => {
       return RequestBank.withdraw('linksTrackingUrl').then(req => {
         const regex = /^\/link\?data=([^&]*)&?.*$/;
         const match = regex.exec(req.url);
@@ -116,8 +115,11 @@ describe('amp-skimlinks', function() {
     });
 
     // TODO(alanorozco): Unskip on firefox
-    // TODO(@slocka) Disabled due to #22154, re-enable
-    it.skip('should send NA-tracking on non-merchant link click ', () => {
+    const itSkipFirefox = (desc, cb) =>
+        it.configure().skipFirefox().run(desc, cb);
+
+    // TODO(alanorozco): Unskip on firefox
+    itSkipFirefox('should send NA-tracking on non-merchant link click ', () => {
       // Give 500ms for amp-skimlinks to set up.
       return browser.wait(500).then(() => {
         clickLinkAndNavigate('#non-merchant-link');
@@ -136,8 +138,7 @@ describe('amp-skimlinks', function() {
       });
     });
 
-    // TODO(@slocka) Disabled due to #22154, re-enable
-    it.skip('Should send merchant links to waypoint on click', () => {
+    it('Should send merchant links to waypoint on click', () => {
       // Give 500ms for amp-skimlinks to set up.
       return browser.wait(500).then(() => {
         clickLinkAndNavigate('#merchant-link');

